README.md

```diff
@@ -50,6 +54,29 @@ sls deploy
 ```
 5. 👀 Head to the CloudWatch section of the AWS Console to check out your new dashboards 📊 and alarms ⏰ !
 
+
+## Getting Started - CloudFormation Macro
+
+1. Deploying to your account (via the console)
+Go to the [AWS Application Repository](https://serverlessrepo.aws.amazon.com/applications) and click the Deploy button.
+
+2. To add the Macro to a SAM template,  add it in the **Transform** section :
+```
+Transform: [ "CfMacroSlicWatch"]
```

Comment:
Can we change this name to match a similar namespaces syntax as other CF macros, e.g., `ORG::SLICWatch::Macro`

Reply:
Throwing the error: Macro name failed for  constraint  [a-zA-Z0-9_-]{1,128}

Reply:
Ok, that's a pity. Can we go for `SLICWatch-Macro` instead?

Reply:
As per the chat discussion, updated the macro name  to "SlicWatch-2022-08-31".  Does this look okay?